syft/workers/base.py

```diff
@@ -375,6 +375,15 @@ def execute_command(self, message: tuple) -> PointerTensor:
 
             response = command(*args, **kwargs)
 
+            # Temporary fix for websockets when returning a tuple of tensors from an LSTM cell
+            if command_name == "torch.lstm_cell":
+                response = torch.stack(response)
+
+            # Temporary fix for websockets when returning a tuple of tensors from torch.sort()
+            if command_name == "torch.sort":
+                Alpha_Tensor_Fixed = (response[0].float(), response[1].float())
+                response = torch.stack(Alpha_Tensor_Fixed)
+
```

Comment:
Same thing, this is a bit hacky and should not be merged in the code base. I understand that there is a pb, but let's open an Issue about this case when we use function with > 1 output and websockets

Reply:
Okay. I guess this would better fit an issue too.